tools/glyph_script_demo.py

```
@@ -0,0 +1,179 @@
+        d.text((frame.width - d.textlength(prog, font=sub_font) - 14, board.height + 14),
+               prog, font=sub_font, fill=(120, 120, 120))
+        imgs.append(frame)
+        durations.append(args.first_hold if i == 0 else args.settle)
+
+    if args.scale != 1.0:
+        size = (int(imgs[0].width * args.scale), int(imgs[0].height * args.scale))
+        imgs = [im.resize(size, Image.LANCZOS) for im in imgs]
+    if args.still:
+        png = os.path.splitext(args.out)[0] + '_still.png'
+        os.makedirs(os.path.dirname(os.path.abspath(png)), exist_ok=True)
+        imgs[0].save(png); print(f"  wrote {png}")
```

Ruff (0.15.21)

[error] Multiple statements on one line (semicolon)

(E702)